添加 wrap mode 变更说明 (#1741)
cocos-creator/creator-docs: en/asset/texture.md

> | __wrapS__ | S (U) direction texture addressing mode |

Reviewer: ```suggestion
| __Wrap Mode S__ | S (U) direction texture addressing mode |
```

> | __wrapT__ | T (V) direction texture addressing mode |

Reviewer: ```suggestion
| __Wrap Mode T__ | T (V) direction texture addressing mode |
```

> > __Note__: Since the default wrap mode may result in black edges when rendering transparent-edge images, we will change the __wrapS__ and __wrapT__ of texture to __clamp-to-edge__ when setting the image resource type to __sprite-frame__, which can be modified by the user if there is a special need.

Reviewer 2: ```suggestion
> __Note__: since the default wrap mode may result in black edges when rendering transparent-edge images, change the __wrapS__ and __wrapT__ of texture to __clamp-to-edge__ when setting the image resource type to __sprite-frame__, which can be modified by the user if there is a special need.
```

> > __Note__: Since the default wrap mode may result in black edges when rendering transparent-edge images, we will change the __wrapS__ and __wrapT__ of texture to __clamp-to-edge__ when setting the image resource type to __sprite-frame__, which can be modified by the user if there is a special need.

Reviewer: ```suggestion
> __Note__: since the default **Wrap Mode** may result in black edges when rendering transparent edges of an image, Creator will change the __Wrap Mode S__ and __Wrap Mode T__ properties of the texture resource to __clamp-to-edge__ automatically when setting the image resource type to __sprite-frame__, which can be modified by the developer if there is a special need.
```
